pkg/crc/cache/cache.go

```diff
@@ -123,3 +130,13 @@ func (c *Cache) CheckVersion() error {
 	}
 	return nil
 }
+
+func IsTarball(filename string) bool {
+	tarballExtensions := []string{".tar", ".tar.gz", ".tar.xz", ".zip", ".tar.bz2", ".crcbundle"}
+	for _, extension := range tarballExtensions {
+		if strings.HasSuffix(strings.ToLower(path.Base(filename)), extension) {
```

Comment:
You can look at `filepath.Ext` also.

Reply:
yes I think we can use the filepath.Ext

Reply:
For `.tar.gz`/`.tar.xz`/`.tar.bz2`, `filepath.Ext` is not going to do the right thing.

Reply:
we can just check `.gz` `.xz` and `.bz2` extensions instead using fully `.tar.gz` right?

Reply:
`foo.gz` is a single file compressed with gzip, `foo.tar.gz` is a tar archive (multiple files) compressed with gzip, passing `foo.gz` to `uncompress` would fail with `"Unknown file format when trying to uncompress %s"`